sdk/ml/azure-ai-ml/azure/ai/ml/operations/_index_operations.py

```diff
@@ -252,6 +253,7 @@ def cls(rest_indexes: Iterable[RestIndex]) -> List[Index]:
         return self._azure_ai_assets.indexes.list(name, list_view_type=list_view_type, cls=cls, **kwargs)
 
     # pylint: disable=too-many-locals
+    @experimental
```

Comment:
```suggestion
```

We already signal to users that the entire IndexOperations class is experimental:

https://github.com/Azure/azure-sdk-for-python/blob/66d5de409d0980d3b2c7b82971cc648f9d5cd6a5/sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py#L1084-L1091

Reply:
Applied